_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+- We recommend using the contrast ratios for icons or other critical elements.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption: "Helix typography styles with their corresponding contrast ratios. All ratios were calculated using #fafafa (gray 25) background color."] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-typography.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Use headings and spacing to group related content
+
+The use of space in visual design provides for designs that are both clear and pleasing to the eyes. Space creates relationships, defines hierarchies, and emphasizes content. Always keep these in mind when considering how to reduce visual noise and create visual flow in your design. Some people with cognitive disabilities find it difficult to read text when the lines are close together. Providing extra space between lines and paragraphs allows them to better track the next line and to recognize when they have reached the end of a paragraph. WCAG prescribes that line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Helix minimum spacing standards require: 
+
+- Line spacing for Roboto Regular 16px is 24px.
+- Paragraph spacing is at least 36px. 
+- Use whitespace and proximity to make relationships between content more apparent.
+- Style headings to group content, reduce clutter, and make it easier to scan and understand.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "Little spacing and unclear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "More spacing and clear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Provide easily identifiable feedback
+
+Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and suggestions to help fix issues whenever errors are returned must be provided. Present any important feedback that requires user action in a prominent style.
```

Comment:
Suggestion: s/`Present any important feedback that requires user action in a prominent style.`/`Prominently present any important feedback that requires user action.`

Reply:
updated.